Doc/library/pprint.rst

```diff
@@ -60,6 +60,9 @@ The :mod:`pprint` module defines one class:
    .. versionchanged:: 3.8
       Added the *sort_dicts* parameter.
 
+   .. versionchanged:: 3.9
```

Comment:
I think it would be better to move it above, just after "Dictionaries are sorted by key before the display is computed."